prime-router/src/testIntegration/kotlin/transport/RESTTransportIntegrationTests.kt

```diff
@@ -387,9 +383,8 @@ class RESTTransportIntegrationTests : TransportIntegrationTests() {
         every { mockRestTransport.lookupDefaultCredential(any()) }.returns(
             UserApiKeyCredential("flexion", "123")
         )
-        every { runBlocking { mockRestTransport.getAuthTokenWithUserEtor(any(), any(), any(), any()) } }.returns(
-            TokenInfoEtor("878a0e79-0e97-4f06-8c44-d756b74e8134", "reportStream",
-                "036231e0-5a1a-4ded-b784-2c4212e311bb")
+        every { runBlocking { mockRestTransport.getAuthTokenWithUserApiKey(any(), any(), any(), any()) } }.returns(
+                TokenInfo("MockToken", null, null, null, "bearer")
```

Comment:
We can use the default null values for the middle 3 parameters
```suggestion
                TokenInfo(accessToken = "MockToken", tokenType = "bearer")
```

Reply:
Ok, I will change it.  I wonder why since they are defaulting to null already in the TokenInfo object?

Reply:
Functionally it doesn't change anything, but I find it a lot more readable and less error-prone to use the parameter names and omit variables that use the defaults. But it's not critical.

Reply:
Done